recipes/sdl/all/conanfile.py

```diff
@@ -192,6 +192,10 @@ def _configure_cmake(self):
                 self._cmake.definitions["PULSEAUDIO"] = self.options.pulse
                 self._cmake.definitions["SNDIO"] = self.options.sndio
                 self._cmake.definitions["NAS"] = self.options.nas
+                if self.options.nas:
+                    self._cmake.definitions["EXTRA_LDFLAGS"] = "-lXau"  # Library files don't propagate transitive deps!
```

Comment:
Investigating together with @SSE4 (thanks!), it looks like the original library ignores the transitive dependencies of `nas`... Have a look here: https://github.com/libsdl-org/SDL/blob/77c8d111370b1dd0d62371c484c95468e645f3d7/cmake/sdlchecks.cmake#L295